app/src/main/java/com/stevesoltys/seedvault/metadata/Metadata.kt

```diff
@@ -17,7 +17,7 @@ data class BackupMetadata(
     internal var time: Long = 0L,
     internal val androidVersion: Int = Build.VERSION.SDK_INT,
     internal val androidIncremental: String = Build.VERSION.INCREMENTAL,
-    internal val deviceName: String = "${Build.MANUFACTURER} ${Build.MODEL}",
+    internal val deviceName: String = "D2D",
```

Comment:
This causes the UI to display the backups names as D2D. We should make the UI portion use `${Build.MANUFACTURER} ${Build.MODEL}`

Reply:
I guess there's something in AOSP that requires this to be D2D exactly? If so, is there a constant for it?

Reply:
https://cs.android.com/android/platform/superproject/+/master:frameworks/base/services/backup/java/com/android/server/backup/restore/ActiveRestoreSession.java;l=302?q=getBackupEligibilityRules

Reply:
> TODO(b/182986784): Remove device name comparison once a designated field for operation type is added to RestoreSet object.

That sounds like it's related to https://github.com/seedvault-app/seedvault/issues/165#issuecomment-938045174

Reply:
This has a TODO that they plan to do this differently. Maybe we are too early with this.

> We should make the UI portion use ${Build.MANUFACTURER} ${Build.MODEL}

How about not changing the metadata info at all, but setting the restore set name/device to D2D instead?

Reply:
Sounds good to me